src/engine/Cache.class.js

```diff
@@ -421,36 +428,78 @@ class Cache {
       const archivedFilename = path.basename(filePath)
       const archivePath = path.join(this.archiveFolder, archivedFilename)
 
-      switch (this.archiveMode) {
-        case 'delete':
-          // Delete original file
-          fs.unlink(filePath, (unlinkError) => {
-            if (unlinkError) {
-              this.logger.error(unlinkError)
-            } else {
-              this.logger.info(`File ${filePath} deleted`)
-            }
-          })
-          break
-        case 'archive':
-          // Create archive folder if it doesn't exist
-          if (!fs.existsSync(this.archiveFolder)) {
-            fs.mkdirSync(this.archiveFolder, { recursive: true })
+      if (this.archiveMode) {
+        // Create archive folder if it doesn't exist
+        if (!fs.existsSync(this.archiveFolder)) {
+          fs.mkdirSync(this.archiveFolder, { recursive: true })
+        }
+
+        // Move original file into the archive folder
+        fs.rename(filePath, archivePath, (renameError) => {
+          if (renameError) {
+            this.logger.error(renameError)
+          } else {
+            this.logger.info(`File ${filePath} moved to ${archivePath}`)
           }
+        })
+      } else {
+        // Delete original file
+        fs.unlink(filePath, (unlinkError) => {
+          if (unlinkError) {
+            this.logger.error(unlinkError)
+          } else {
+            this.logger.info(`File ${filePath} deleted`)
+          }
+        })
+      }
+    }
+  }
 
-          // Move original file into the archive folder
-          fs.rename(filePath, archivePath, (renameError) => {
-            if (renameError) {
-              this.logger.error(renameError)
-            } else {
-              this.logger.info(`File ${filePath} moved to ${archivePath}`)
-            }
-          })
-          break
-        default:
-          this.logger.error(`unknown Archive Mode: ${this.archiveMode}`)
+  /**
+   * Delete file in archiveFolder
+   * @param {string} filePath - The file
+   * @return {void}
+   */
+  async refreshArchiveFolder() {
+    this.logger.silly('refreshArchiveFolder()')
```

Comment:
This log is a little bit dirty. Can you rephrase it ? Something like 'Looking for archived files to remove'
And change it from silly to debug